src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -230,19 +231,20 @@ private ServerConnectionProps getServerProps(final Component ui) {
       } else {
         JOptionPane.showMessageDialog(ui, "Invalid Port: " + port, "Error", JOptionPane.ERROR_MESSAGE);
       }
-      return null;
+      return Optional.empty();
     }
     final ServerConnectionProps props = new ServerConnectionProps();
     props.setName(options.getName());
     props.setPort(options.getPort());
     props.setPassword(options.getPassword());
-    return props;
+    return Optional.of(props);
   }
 
 
   public boolean createServerMessenger(final Component ui) {
```

Comment:
Checkstyle: "Missing a Javadoc comment."

Due to increasing the method line count from two to three.

Reply:
Javadoc'ed the method in: c98473e